pkgs/servers/ldap/lldap/default.nix

```diff
@@ -20,28 +20,31 @@ let
 
   commonDerivationAttrs = rec {
     pname = "lldap";
-    version = "0.5.0";
+    version = "0.5.1-alpha-6aa930";
```

Comment:
```suggestion
    version = "0.5.0-unstable-2024-08-06";
```

Reply:
I can see the timestamp instead of commit hash. But as for the version number, lldap on `main` designates itself `0.5.1-alpha` in the lock file. Shouldn't we follow that in this case?

Reply:
The format we decided to use depends on the last released tag plus the date, so we don't need to check if upstream decided to call their devel version something something.

Reply:
ok, is this in an RFC somewhere?